conda_build_all/version_matrix.py

```diff
@@ -221,9 +217,24 @@ def add_case_if_soluble(case):
                 add_case_if_soluble(case)
 
         if 'perl' in requirement_specs:
-            raise NotImplementedError('PERL version matrix not yet implemented.')
+            pl_spec = requirement_specs.pop('perl')
+            for case_base in list(cases or [()]):
+                for perl_pkg in r.get_pkgs(pl_spec):
+                    pl_vn = index[perl_pkg.fn]['version']
+                    case = case_base + (('perl', pl_vn), )
+                    add_case_if_soluble(case)
+                if case_base in cases:
+                    cases.remove(case_base)
+
         if 'r' in requirement_specs:
-            raise NotImplementedError('R version matrix not yet implemented.')
+            r_spec = requirement_specs.pop('r')
```

Comment:
Not perfect, as it duplicates the same logic as for perl, but good enough for now...